internal/pkg/model/defaultbucket.go

```diff
@@ -10,7 +10,7 @@ type UsedInInputMappingRelation struct {
 }
 
 func (t *UsedInInputMappingRelation) Type() RelationType {
-	return UsedInOrchestratorRelType
+	return UsedInInputMappingRelType
```

Comment:
@JakubMatejka tu si mal preklep.

Reply:
Omg 🙈 děkuju.

Reply:
Právě jsem tam hledal, myslel jsem že by to mohlo být něco takovýho, ale už jsem to asi pro oči neviděl. 😅